crypto/bf/build.info

```diff
@@ -17,7 +17,7 @@ SOURCE[../../libcrypto]=$ALL
 
 # When all deprecated symbols are removed, libcrypto doesn't export the
 # blowfish functions, so we must include them directly in liblegacy.a
-IF[{- $disabled{'deprecated-3.0'} -}]
+IF[{- $disabled{'deprecated-3.0'} && !$disabled{module} && !$disabled{shared} -}]
```

Comment:
Not sure I understand the justification for "shared" here. If we build "no-shared" then we still build the legacy provider module don't we?

Reply:
If we build with no-shared, the legacy provider will still be a shared module, however it will be statically linked to the libcrypto.a because there is no shared libcrypto. These symbols will be available from libcrypto.a even with no-deprecated.